src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupUtilities.cs

```diff
@@ -136,6 +147,109 @@ public BackupConfigInfo GetBackupConfigInfo(string databaseName)
         }
 
         /// <summary>
+        /// Execute backup
+        /// </summary>
+        public void PerformBackup()
+        {
+            this.backup = new Backup();
+            this.SetBackupProps();
+            backup.Database = this.backupInfo.DatabaseName;
+            backup.Action = this.backupActionType;
+            backup.Incremental = this.IsBackupIncremental;
+            if (backup.Action == BackupActionType.Files)
+            {
+                IDictionaryEnumerator IEnum = this.backupInfo.SelectedFileGroup.GetEnumerator();
+                IEnum.Reset();
+                while (IEnum.MoveNext())
+                {
+                    string CurrentKey = Convert.ToString(IEnum.Key,
```

Comment:
>CurrentKey [](start = 27, length = 10)

...camelCase variables names please...I won't list any more, but please update all variables to follow naming conventions for this code base.